datamodel/datamodel.go

```diff
@@ -0,0 +1,9 @@
+/*
+Package datamodel defines constants for the various back-end storage mechanisms supported by hegel.
+*/
+package datamodel
+
+const (
+	TinkServer = "1"
+	Kubernetes = "kubernetes"
+)
```

Comment:
Is it worth a type alias?
```suggestion

type DataModel string

const (
	DataModelTinkServer = DataModel("1")
	DataModelKubernetes = DataModel("kubernetes")
)
```

Reply:
Yes

Reply:
Will add to the other PR

Reply:
Reworked PR stack so now done here.